python/GangaDirac/test/Unit/Utilities/TestDiracUtilitiesExternals.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
There's no need to split these out into a separate file. It probably makes more sense to keep them all in one place (I do agree though that it makes sense to split out some of these tests into parts since some of it calls DIRAC and other parts don't).